site/cypress/e2e/Cypress-UI/pronouns_change.spec.js

```diff
@@ -77,8 +77,8 @@ describe('Tests cases abut changing user pronouns', () => {
         // verify pronouns is shown in overall forum page
         cy.get('#nav-sidebar-forum').click();
         cy.get('.thread-list-item').should('contain', 'Test pronouns display');
-        cy.contains('Test pronouns display').find('.post_user_pronouns').should('contain', 'They/Them');
-        cy.contains('Test pronouns display').find('.post_user_pronouns').click();
+        cy.get('.thread_box.thread-box-full').contains('Test pronouns display').parent().find('.post_user_pronouns').should('contain', 'They/Them');
```

Comment:
We should eventually refactor at least some of this file to use data-testids. Either you can add them here instead of using html selectors (.xxx or #xxx), or create an issue and have it changed in the near future. I don't usually check out the branch to try the test on the first round of reviews, and I just read through the code to see if there is anything that should be done differently (best practice, or an easier way to do something, etc), and it is difficult to do that on this file as it was created before we started emphasizing testids.